dll/win32/shimgvw/shimgvw.c

```diff
@@ -597,6 +713,12 @@ ImageView_WndProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
         }
         break;
 
+        case WM_MOUSEWHEEL:
+            ImageView_OnMouseWheel(hwnd,
+                (SHORT)LOWORD(lParam), (SHORT)HIWORD(lParam),
+                (SHORT)HIWORD(wParam), (UINT)LOWORD(wParam));
```

Comment:
I would strongly recommend here the usage of: `GET_X_LPARAM(lParam);` and `GET_Y_LPARAM(lParam);` for the X and Y deltas.

Reply:
Done.